cluster/gce/util.sh

```diff
@@ -1362,7 +1362,7 @@ function kube-down() {
     "${NETWORK}-default-ssh" \
     "${NETWORK}-default-internal"  # Pre-1.5 clusters
   if [[ "${KUBE_DELETE_NETWORK}" == "true" ]]; then
-    delete-network
+    delete-network || true  # might fail if there are leaked firewall rules
```

Comment:
@krzyzacy is working on a janitor to clean up leaked resources, so I'm not super worried about this.